src/store/metadata/stub/content/topics.json

```diff
@@ -24,6 +24,28 @@
             "parent_topics": [
                 "https://staging.idpd.uk/topics/economy"
             ]
+        },
+        {
+            "@id": "https://staging.idpd.uk/topics/emissions",
+            "@type": "dcat:theme",
+            "identifier": "emissions",
+            "title": "Emissions",
+            "description": "something describing emissions",
+            "sub_topics": [
+                "https://staging.idpd.uk/topics/greenhouse-gas"
+            ],
+            "parent_topics": null
+        },
+        {
+            "@id": "https://staging.idpd.uk/topics/greenhouse-gas",
+            "@type": "dcat:theme",
+            "identifier": "greenhouse-gas",
+            "title": "Greenhouse Gas",
+            "description": "something describing greenhouse-gas",
+            "sub_topics": null,
```

Comment:
this was perfect when you did it but we went and changed something very slightly, instead of `null` can you use an empty list please, i.e `[]`.